feature/src/main/java/com/terning/feature/intern/InternViewModel.kt

```diff
@@ -28,11 +53,14 @@ class InternViewModel @Inject constructor() : ViewModel() {
     }
 
     fun updateScrapped(scrapped: Boolean) {
-        _state.value = _state.value.copy(isScrapped = scrapped)
+        _state.value = _state.value.copy(isScrappedState = scrapped)
     }
 
     fun updateColorChange(change: Boolean) {
         _state.value = _state.value.copy(isColorChange = change)
     }
 
+    fun updateShowWeb(show: Boolean) {
+        _state.value = _state.value.copy(showWeb = show)
+    }
```

Comment:
```suggestion
    fun updateShowWeb(show: Boolean) {
        _state.update{ currentState ->
            currentState.copy(showWeb = show)
        }
    }
```
과 같이 값을 업데이트하는 방법이 있다는데 참고해봐요~
아토믹하게 값을 업데이트하는 방법이라고 배웠어요!

Reply:
와 이건 왕따봉 드림여 ㅜㅜ